mainPlugin/src/main/java/fr/il_totore/manadrop/spigot/task/BuildTools.java

```diff
@@ -22,10 +22,13 @@
     public boolean showMavenInstallCheckLogs = false;
     public boolean showBuildToolsLogs = true;
     public int maxRamAllowed = 1024;
-    public String mavenPath = System.getenv("MAVEN_HOME") + "/bin/mvn" + (System.getenv("OS") != null && System.getenv("OS").toLowerCase().contains("win") ? ".cmd" : "");
+    public String mavenPath = System.getProperty("user.home") + ".m2/repository/";
```

Comment:
This doesn't solve anything. What if you're maven installation is not in `~/.m2/repository` also this is the **repository** dir. Not the executable's one.

Reply:
Fixed in new commit